mains/mains for meat lover

```diff
@@ -0,0 +1 @@
+we should add some mains food for non-vegetarians.
```

Comment:
For example, what about the beef hamburger?

Reply:
Sounds great

Reply:
you should check out the typo on line 12 about the 2 cup(s)